honeybee_schema/radiance/properties.py

```diff
@@ -0,0 +1,176 @@
+"""Properties Schema"""
+from pydantic import Field, constr, validator, root_validator
+from typing import List, Union
+
+from .._base import IDdBaseModel
+from .modifier import Void, Plastic, Glass, BSDF, Glow, Light, Trans, ModifierSet
+
+
+class _PropertiesBaseAbridged(IDdBaseModel):
+    """Base class of Abridged Radiance Properties."""
+
+    modifier: str = Field(
+        min_length=1,
+        max_length=100,
+        description='A string for a Honeybee Radiance Modifier.'
+        )
+
+    modifier_blk: str = Field(
+        min_length=1,
+        max_length=100,
+        description='A string for a Honeybee Radiance Modifier to be used '
+                    'in direct solar simulations and in isolation studies (assessing'
+                    'the contribution of individual objects).'
+        )
+
+
+class _PropertiesBase(IDdBaseModel):
+    """Base class of Radiance Properties."""
+
+    modifier: Union[
+         Void, Plastic, Glass, BSDF, Glow, Light, Trans
+    ] = Field(
```

Comment:
This list will get larger and larger over time. The way it is implemented right now will be limiting to what we need to handle the cases with other types of material.

Two suggestions here:

1. Create this list as an object that can be re-used to avoid updating it in several places in the future.
2. Create a base class and subclass all the other Radiance modifiers from the baseclass for now similar to how they are implemented in honeybee-radiance. Otherwise I can see this to be a problem as soon as people start using the schema.

Reply:
@mostaphaRoudsari 

I think I can do option 2 then, I'm already subclassing all Radiance modifiers with a `ModifierBase` class.

Reply:
Implemented.